source/how-we-work/version-control/support-branches.html.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Support branches
+weight: 2
+---
+
+# Support branches
+
+We use support branches when we want to:
+
+- publish a new release of previous major versions of GOV.UK Frontend
+
+- publish a 'hotfix' release of GOV.UK Frontend without including other unreleased changes on the `main` branch
+
+Once a support branches has been created it should be maintained in parallel to the `main` branch. The support branch will never be merged into the `main` branch.
+
+Support branches have the same branch protection rules as the main branch. Any changes made in a support branch should be raised as Pull Requests.
+
+## Using support branches for previous major versions
+
+We can use support branches to publish new releases of a previous major version.
+
+For example, if we found a bug that affected all versions of GOV.UK Frontend between v4.6.0 and v5.1.0, we might choose to release a new version of v4 as well as a new version of v5 so that users stuck on v4 can get the fix without needing to upgrade to v5.
+
+We might also use a support branch so that we can continue to release when we have unreleased breaking changes on the main branch during the initial development of a new major version.
+
+![Graph showing the main and support branch running in parallel. The support/4.x branch diverges from the last v4 tag on the main branch. After the point the two branches diverge, the main branch has tags for v5.0.0 and v5.0.1 and the support branch has tags for v4.6.1 and v4.6.2.](/images/support-branch-prev-major.svg)
+
+<!-- 
+gitGraph
+    commit
+    commit tag: "v4.6.0"
+    branch "support/4.x"
+    checkout "main"
+    commit
+    commit
+    commit
+    commit tag: "v5.0.0"
+    commit
+    checkout "support/4.x"
+    commit
+    commit tag: "v4.6.1"
+    checkout "main"
+    commit
+    commit
+    commit tag: "v5.0.1"
+    checkout "support/4.x"
+    commit
+    commit tag: "v4.6.2"
+-->
+
+There should only be one support branch for each previous major version. The support branch should be named using only the major version number, for example the branch name for the v4 major release would be called `support/4.x`.
+
+Because the support branch will never be merged back into the main branch, any changes that affect both current and previous major versions require multiple pull requests -- one for each support branch for each of the affected previous major versions, and one to fix it on the main branch for the current major version.
```

Comment:
Minor content suggestion because this is a bit of a mouthful:

```suggestion
Because the support branch will never be merged back into the main branch, any changes that affect both current and previous major versions require multiple pull requests -- one per support branch for each of the affected previous major versions, and one to fix it on the main branch for the current major version.
```